README.md

```diff
@@ -1,4 +1,4 @@
-# air-traffic-control
+# Correlating Flight and Weather Data in Augmented Reality
 [![Build Status](https://travis-ci.org/IBM/air-traffic-control.svg?branch=master)](https://travis-ci.org/IBM/air-traffic-control)
```

Comment:
This project is more than just correlating flight and weather data in Augmented Reality. It has a Raspberry Pi powered Ground Station that uses a Software Defined Radio to receive ADS-B messages, decode the ADS-B messages to retrieve flight information, and then publishes MQTT messages with flight information in the JSON payload over to IoT Platform. In effect, we are diluting the most important aspect of this project if we say that it is just `Correlating Flight and Weather Data in Augmented Reality`. Please contact me if you have any questions about this.

Reply:
If use Software Defined Radio, can it be done to be replaced by implementing software on Rasberry devices or embedded systems of computer mother boards.